examples/typescript/Anchor.toml

```diff
@@ -1,3 +1,7 @@
 [provider]
 cluster = "localnet"
 wallet = "~/.config/solana/id.json"
+
+[programs]
```

Comment:
What do you think of renaming this to `[workspace]`, similar to Cargo?

I'm a little hesitant to use the `[programs]` name because in the future we may want to combine the `[[test.genesis]]` and `[clusters.<network>]` sections into one (both allow customizing various features of programs in the workspace).

Reply:
I think it's will be better, thanks for the suggestion! Changed. But now I think, should `[workspace]` have `members` & `exclude` as Cargo (with `members` default to `programs/*`)?

Reply:
Sounds good to me! `members` may be better since it's consistent with Cargo but I'm happy with either choice.

Reply:
Went ahead and did the rename.

Reply:
Thanks. I thought to add regex support but was busy yesterday.

Reply:
Would be cool to see in a subsequent PR.

Reply:
Sure. Should we use default value for `members` as`["programs/*"]`?

Reply:
Let's hold off on the regex support. I'm making some changes that will conflict with this.

Reply:
To support package publishing, I've changed the `members` and `exclude` arrays to take paths relative to the workspace root (instead of relative to the `programs/` dir, as is done in this PR), which is consistent with how Cargo handles this.  

The next version will have these changes in it. But the immediate release following will contain the breaking change (PR here https://github.com/project-serum/anchor/pull/570).